toolbox/pkgs/emitter/templates/src-libvfn.sh.jinja

```diff
@@ -0,0 +1,12 @@
+#
+# Clone, build and install libvfn
+#
+# Assumptions:
+#
+# - These commands are executed with sufficient privileges (sudo/root)
+#
+git clone https://github.com/OpenMPDK/libvfn.git
```

Comment:
This should probably checkout the right version of libvfn and not master.

Reply:
Good point, fixed!